src/offlinelogstorage/dlt_offline_logstorage_behavior.c

```diff
@@ -935,22 +1045,29 @@ int dlt_logstorage_write_on_msg(DltLogStorageFilterConfig *config,
         return -1;
     }
 
-    ret = fwrite(data1, 1, size1, config->log);
+    ret = dlt_logstorage_write_to_log(data1, 1, size1, config);
 
     if (ret != size1)
         dlt_log(LOG_WARNING, "Wrote less data than specified\n");
 
-    ret = fwrite(data2, 1, size2, config->log);
-
+    ret = dlt_logstorage_write_to_log(data2, 1, size2, config);
     if (ret != size2)
         dlt_log(LOG_WARNING, "Wrote less data than specified\n");
 
-    ret = fwrite(data3, 1, size3, config->log);
-
+    ret = dlt_logstorage_write_to_log(data3, 1, size3, config);
     if (ret != size3)
         dlt_log(LOG_WARNING, "Wrote less data than specified\n");
 
+    if (config->gzip_compression) {
```

Comment:
Missing close bracket leads to error compiling.
Maybe line 1061 should be included under ifdef?!